libs/core/src/components/sage-checkbox/test/sage-checkbox.e2e.ts

```diff
@@ -0,0 +1,44 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('sage-checkbox', () => {
+  it('toggles checked and unchecked', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<sage-checkbox checkbox-id="default" label="Label text" />');
+    const component = await page.find('sage-checkbox');
+    expect(component).toHaveClass('hydrated');
+    const checkbox = await page.find('sage-checkbox >>> input');
+    await checkbox.click();
+    expect(await checkbox.getProperty('checked')).toBeTruthy();
+    await checkbox.click();
```

Comment:
add a new line here. It will be the same for the rest of this file. Also, we want to have a new line when starting a new test `it(....)`.